editquality/utilities/merge_labels.py

```diff
@@ -54,6 +58,13 @@ def run(human_labels, auto_labels, labels_f, verbose):
                      for revision in human_labels}
     human_rev_ids = set(human_rev_map.keys())
 
+    # Output human labels when autolabels is empty
+    if not auto_labels:
```

Comment:
I'd rather make this a separate function which is called conditionally, but it's fine for now.

Reply:
Good point, sorry I missed it.